app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.java

```diff
@@ -33,7 +32,7 @@
 import static org.mozilla.focus.fragment.FirstrunFragment.FIRSTRUN_PREF;
 
 // This test checks all the headings in the Settings menu are there
-@RunWith(AndroidJUnit4.class)
+@Suppress
```

Comment:
What's the goal here? Let's either fix the test or remove it. If we suppress it for a longer time then it will not work anymore eventually anyways.

Reply:
oh I suppressed it because fix for the SwitchLocaleTest is in other PR, and failing this test will affect all subsequent tests (since it changes Locale).  One of the reason why this PR wasn't ready for the review :)